nemo/collections/asr/models/ctc_models.py

```diff
@@ -335,6 +335,22 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
         else:
             augmentor = None
 
+        is_concat = config.get('is_concat', False)
+        if is_concat:
+            if 'concat_sampling' in config and config['concat_sampling'] is None:
+                logging.warning(
+                    f"Concat dataset requires `contact_sampling` but it was not provided. Config: {config}"
+                )
+                return None
+            if 'concat_probabilities' in config and config['concat_probabilities'] is None:
+                logging.warning(
+                    f"Concat dataset requires `contact_probabilities` but was not provided. Config: {config}"
+                )
+                return None
```

Comment:
Above comments

Reply:
Doable but somewhat tricky; would need to initialize array with values :/

Reply:
Skip then.